dashboard/models.py

```diff
@@ -105,10 +91,15 @@ def delete_all_but(cls, user, course_ids_list):
         Returns:
             None
         """
-        cls.active_qset_all(user).exclude(course_run__edx_course_key__in=course_ids_list).delete()
+        cls.active_qset(user).exclude(course_run__edx_course_key__in=course_ids_list).delete()
 
     @classmethod
-    def get_edx_data(cls, user):
+    def get_raw_data(cls, user, program=None):
+        """Returns a list of 'data' values for all matching objects"""
+        return list(cls.data_qset(user, program=program).all())
```

Comment:
what's the point of making this additional method? I think either `data_qset` or this one are more than enough.